custom/icds_reports/tasks.py

```diff
@@ -631,12 +631,18 @@ def _agg_awc_table_weekly(day):
 
 
 @task(serializer='pickle', queue='icds_aggregation_queue')
-def email_dashboad_team(aggregation_date):
+def email_dashboad_team(aggregation_date, aggregation_start_time):
+    aggregation_start_time = aggregation_start_time + timedelta(hours=5, minutes=30)
+    aggregation_finish_time = datetime.utcnow() + timedelta(hours=5, minutes=30)
```

Comment:
This should be using something more similar to https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/templatetags/icds_format.py#L103

Reply:
DOne